apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/UserInfoController.java

```diff
@@ -75,5 +75,14 @@ public UserInfo getUserByUserId(@PathVariable String userId) {
     return userService.findByUserId(userId);
   }
 
+  /**
+   * 查询用户列表
+   * @return
+   */
+  @GetMapping("/users/list")
+  public List<UserInfo> getUserList() {
```

Comment:
Do we really need a separate API? Does the API below meets the requirement?
```java

  @GetMapping("/users")
  public List<UserInfo> searchUsersByKeyword(@RequestParam(value = "keyword") String keyword,
                                             @RequestParam(value = "offset", defaultValue = "0") int offset,
                                             @RequestParam(value = "limit", defaultValue = "10") int limit) {
    return userService.searchUsers(keyword, offset, limit);
  }
```

such as `/users?keyword=*`?

If not, the API pattern doesn't respect to RESTful style, I suggest using `@GetMapping("/users")` and change the annotation of `searchUsersByKeyword ` to `  @GetMapping(value = "/user/logout", params = {"keyword"})`